readme.md

```diff
@@ -32,6 +32,10 @@ console.log(fallbackSymbols.tick);
 console.log(replaceSymbols('✔︎ check'));
 // On terminals with Unicode symbols:  ✔︎ check
 // On other terminals:                 √ check
+
+console.log(replaceSymbols('✔︎ check', {useFallback: true}));
```

Comment:
This is not a common need, so I prefer to move the example to where the option is defined in the docs.

Reply:
Done in 0f42c12.